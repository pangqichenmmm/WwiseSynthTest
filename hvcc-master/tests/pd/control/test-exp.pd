#N canvas 231 101 305 219 10;
#X obj 35 134 exp;
#X obj 35 33 loadbang;
#X obj 35 55 t b b b b b b;
#X msg 35 97 -100;
#X msg 73 97 -10;
#X msg 105 97 0;
#X msg 137 97 2;
#X msg 171 97 134;
#X msg 207 97 10000;
#X obj 35 166 print;
#X connect 0 0 9 0;
#X connect 1 0 2 0;
#X connect 2 0 3 0;
#X connect 2 1 4 0;
#X connect 2 2 5 0;
#X connect 2 3 6 0;
#X connect 2 4 7 0;
#X connect 2 5 8 0;
#X connect 3 0 0 0;
#X connect 4 0 0 0;
#X connect 5 0 0 0;
#X connect 6 0 0 0;
#X connect 7 0 0 0;
#X connect 8 0 0 0;
