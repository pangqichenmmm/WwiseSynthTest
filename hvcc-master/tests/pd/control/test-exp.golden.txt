[@ 0.000] print: 8.5067e+37
[@ 0.000] print: 8.5067e+37
[@ 0.000] print: 7.38906
[@ 0.000] print: 1
[@ 0.000] print: 4.53999e-05
[@ 0.000] print: 3.78351e-44
